app/qml/AboutPanel.qml

```diff
@@ -20,7 +20,6 @@ import "./components"
 Page {
   id: root
 
-  property string inputLink: "https://inputapp.io/"
```

Comment:
this is efectively "input landing page", for example in future merginmaps.com/input or something like that. It is not "__inputHelp,inputWebLink" since that one would be merginmaps.com/docs/..? something